clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -3665,6 +3670,18 @@ public void testWhenFetchResponseReturnsALeaderShipChangeErrorAndNewLeaderInform
         // Validate subscription is still valid & fetch-able for tp1.
         assertTrue(subscriptions.isFetchable(tp1));
     }
+    
+    @Test
+    public void testFetcherDontCacheAnyData() {
+        short version = 17;
```

Comment:
Do we have any existing test where `version < 13`? If not then can we please add one.

Reply:
@apoorvmittal10, Thanks for your comments, I will add a test for version < 13

Reply:
@apoorvmittal10, Thanks for your comments, In `testFetchWithNoTopicId` have been test for version 12, Should I 
add a test only for `FetchResponse#responseData` this method to test version 12? WDYT

Reply:
Thansk for looking into, if there already exists a test then we can skip new one.